src/components/Divider.tsx

```diff
@@ -36,12 +40,19 @@ const Divider = ({ path, cssRaw }: { path: Path; cssRaw?: SystemStyleObject }) =
 
   useEffect(setStyle)
 
+  const isTableView = useSelector(state => {
+    const parentId = head(parentOf(path)) || head(rootedParentOf(state, path))
+    const result = attributeEquals(state, parentId, '=view', 'Table')
+    return result
```

Comment:
Avoid creating an additional variable here and just return the statement directly.